Library/Formula/vowpal-wabbit.rb

```diff
@@ -17,12 +17,72 @@ class VowpalWabbit < Formula
   depends_on "autoconf" => :build
   depends_on "automake" => :build
   depends_on "libtool" => :build
+  needs :cxx11
 
   def install
+    ENV.cxx11
     ENV["AC_PATH"] = "#{HOMEBREW_PREFIX}/share"
     system "./autogen.sh", "--prefix=#{prefix}",
-                           "--with-boost=#{Formula['boost'].opt_prefix}"
+                           "--with-boost=#{Formula['boost'].opt_prefix}",
+                           "--enable-libc++"
     system "make"
     system "make", "install"
   end
-end
+
+  test do
+    system bin/"vw", "--version"
+    system bin/"vw", "--help"
+
+    (testpath/"house_dataset").write("0 | price:.23 sqft:.25 age:.05 2006
+1 2 'second_house | price:.18 sqft:.15 age:.35 1976
+0 1 0.5 'third_house | price:.53 sqft:.32 age:.87 1924")
+    system bin/"vw house_dataset"
+    system bin/"vw house_dataset --nn 10"
+    system bin/"vw house_dataset --noconstant"
+    system bin/"vw house_dataset --constant 1"
+    system bin/"vw house_dataset --ngram 2 --skips 2"
+    system bin/"vw house_dataset --l1 0.01 --l2 0.01"
+    system bin/"vw house_dataset --loss_function quantile"
+    system bin/"vw house_dataset -l 10 -c --passes 25 --holdout_off --decay_learning_rate=.95"
+    system bin/"vw house_dataset -l 10 -c --passes 25 --holdout_off"
+    system bin/"vw house_dataset -l 10 -c --passes 25 --holdout_off -f house.model"
+    system bin/"vw house_dataset -p /dev/stdout --quiet"
+    system bin/"vw -i house.model -t house_dataset -p /dev/stdout --quiet"
+    system bin/"vw house_dataset --audit --quiet"
```

Comment:
Probably only need a single `vw` invocation on some data and `"separate", "strings", "like", "this"`